lib/rubocop/cop/flag/feature_flag_name.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Flag
+      # The Feature Flag Name Cop is used to identify flag names that are not
+      # using the sneak case pattern.
+      #
+      # @example
+      #   # bad
+      #   if FeatureFlag.enabled? 'SEARCH_USE_DECRYPTED_RESPONDENT_ID'
+      #
+      #   # good
+      #   if FeatureFlag.enabled? 'search_use_decrypted_respondent_id'
+      class FeatureFlagName < Base
+        def_node_matcher :feature_flag_name, <<~PATTERN
+        (send
+          (const nil? :FeatureFlag) :enabled?
+          (str $_))
+        PATTERN
+
+        RESTRICT_ON_SEND = [:enabled?].freeze
+
+        def on_send(node)
+          feature_flag_name(node) do |name|
+            next if to_snake_case(name) == name
+
+            add_offense(node, message: 'Use snake_case for feature flag names.')
```

Comment:
can it mention on the error message, that it also need to be lowercase?

Reply:
Since SCREAMING_SNAKE_CASE can be considered snake case as well 😆

Reply:
Good catch! ❤️